config/initializers/preload_working_days.rb

```diff
@@ -3,5 +3,5 @@
 # Don't preload the bank_holidays in test, so that we can stub out the calls in test setup methods.
 unless Rails.env.test?
   # This loads the bank holiday dates from https://www.gov.uk/bank-holidays.json
-  WorkingDays.bank_holidays
+  WorkingDays.send(:bank_holidays)
```

Comment:
Why is this change necessary?

Reply:
As part of [this commit](https://github.com/alphagov/smart-answers/pull/3719/commits/d7ad47aa9d78cc492054b4d0f1ec3910fa56590a) I made the `self.bank_holidays` method private. It looks like the intent was for `bank_holidays` to be a private method, but as a singleton method it was still accessible. Since the method is now private I needed to use `.send` here so that I could bypass the private check.

Let me know if it would be better to make the method public. I can also undo this change, and add in a `rubocop:disable` comment.